language/move-lang/functional-tests/tests/0L/reconfiguration/reconfig_case_2_rejoin.move

```diff
@@ -227,7 +227,7 @@ script {
         Vector::push_back<address>(&mut voters, @{{bob}});
         Vector::push_back<address>(&mut voters, @{{carol}});
         Vector::push_back<address>(&mut voters, @{{dave}});
-        // Vector::push_back<address>(&mut voters, @{{eve}});
```

Comment:
@0o-de-lally when I made the change, this test started failing the assert in line 252, eve was hitting case 4 rather than case 2. Case 2 means eve was validating but not mining, but with this line commented out, Eve was not validating, so case 4 would be correct. Am I missing something here?

Reply:
@jamesmeijers  Yes this test needs to be cleaned up. The text comments refer to EVE as the Case 2. But we're actually testing FRANK. So your change is correct (but I'm not sure why it would have been passing before your change).

The check on line 242 is not necessary, since at that point in the test all validators will be case 2 (since we don't mock the mining yet).

I also noticed that the two scripts in 94 and 116 are duplicates. I think it's harmless though.

Reply:
Glad to hear my understanding is right, very odd/concerning that is was passing before though.